src/neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -44,30 +37,7 @@ protected override bool OnPersist(ApplicationEngine engine)
             ECPoint[] validators = NEO.GetNextBlockValidators(engine.Snapshot);
             UInt160 primary = Contract.CreateSignatureRedeemScript(validators[engine.Snapshot.PersistingBlock.ConsensusData.PrimaryIndex]).ToScriptHash();
             Mint(engine, primary, engine.Snapshot.PersistingBlock.Transactions.Sum(p => p.NetworkFee));
-            BigInteger sys_fee = GetSysFeeAmount(engine.Snapshot, engine.Snapshot.PersistingBlock.Index - 1) + engine.Snapshot.PersistingBlock.Transactions.Sum(p => p.SystemFee);
```

Comment:
Also we should decrement the `Total Supply` otherwise people won't know the inflation

Reply:
I need to re-check maybe it's done already here https://github.com/neo-project/neo/pull/1430/files#diff-721ebd5f13602811b4f33d4a51a354d7R36